src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -92,6 +94,14 @@ public Response getZipDownloadLimit() {
         return ok(zipDownloadLimit);
     }
 
+    @GET
+    @Path("exportFormats")
+    public Response getExportFormats() {
+        JsonArrayBuilder responseModel = Json.createArrayBuilder();
+        ExportService.getInstance().getExportersLabels().forEach(labels -> responseModel.add(Json.createObjectBuilder().add("displayName", labels[0]).add("formatName", labels[1])));
+        return ok(responseModel);
+    }
+
     private Response getSettingResponseByKey(SettingsServiceBean.Key key) {
         String setting = settingsService.getValueForKey(key);
         if (setting != null) {
```

Comment:
I'll just put this comment at the bottom. An API test would be nice. I can help you get started if you need it.

Reply:
A basic API test has been added in [95d34c8](https://github.com/IQSS/dataverse/pull/10739/commits/95d34c8704367dd982906934f812b28d327874b4), which compares the response to what should be expected from a Dataverse installation with the default set of exporters. Is this what you had in mind?

Reply:
Yes! I left a comment on the test. Thanks!